### PR TITLE
Initial code

### PR DESCRIPTION
**Pull request by archive64**

### CODE REVIEW

File: src/demo.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  property: 'company',
+  label: 'Company'
+}, {
+  property: 'sentence',
+  label: 'Sentence'
+}];
+
+class SpecificWikiTableEditor extends React.Component {
+  constructor(props) {
+    super(props);
```
Comment (copperwall) [inline src/demo.jsx:51]: super props, bro

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "react-dnd": "^2.1.4",
+    "react-dnd-html5-backend": "^2.1.2",
+    "react-dom": "^15.4.1",
+    "react-edit": "^6.1.1",
+    "reactabular": "^8.3.0",
+    "reactabular-dnd": "^8.4.1",
+    "reactabular-table": "^8.5.0",
+    "redux": "^3.6.0",
+    "table-resolver": "^2.0.3",
+    "webpack": "^1.14.0"
```
Comment (solflare) [inline package.json]: Do we want to lock these down a bit more?

Comment (archive64) [reply package.json]: Sure: https://github.com/iFixit/wiki-table-editor/pull/1/commits/26b85670a105a3425d5543d224bc87d63f97f190

File: package.json [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
+  },
+  "devDependencies": {
+    "babel-core": "6.21.0",
+    "babel-loader": "6.2.10",
+    "babel-plugin-jsx": "1.2.0",
+    "babel-plugin-transform-object-rest-spread": "6.20.2",
+    "babel-preset-es2015": "6.18.0",
+    "babel-preset-es2016": "6.16.0",
+    "babel-preset-react": "6.16.0",
+    "webpack": "1.14.0"
```
Comment (solflare) [inline package.json:22]: Very nice :+1:

**Comment by copperwall** [comment]
CR :fire:

**Comment by solflare** [comment]
Dang I thought I cr'd this CR :moon:

**Pull request closed without merging**